otter/test/convergence/test_service.py

```diff
@@ -857,7 +858,7 @@ def setUp(self):
         self.now = datetime(1970, 1, 1)
         self.waiting = Reference(pmap())
 
-    def get_seq(self, with_cache=True):
+    def get_seq(self, with_cache=True, upd_status_nogroup_err=False):
```

Comment:
That parameter's role is far from obvious to the casual reader: this needs a docstring :)